sonic-chassisd/tests/test_chassisshow.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+from imp import load_source
+
+from mock import Mock, MagicMock, patch
+from sonic_py_common import daemon_base
+
+from .mock_platform import MockChassis, MockModule
+from .mock_module_base import ModuleBase
+
+SYSLOG_IDENTIFIER = 'chassisshow_test'
+NOT_AVAILABLE = 'N/A'
+
+daemon_base.db_connect = MagicMock()
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+sys.path.insert(0, modules_path)
+
+os.environ["CHASSISSHOW_UNIT_TESTING"] = "1"
+load_source('chassisshow', scripts_path + '/chassisshow')
+from chassisshow import *
+
+
+def test_provision_db():
+    chassis = MockChassis()
+    log = MagicMock()
+    serial = "Serial No"
```

Comment:
Isn't it better to create the text of each as constant and use the same in different files?
Here it is written in 2 different places. not must but i think it is better.

Reply:
It might be, although if you check `sonic-chassisd/tests/test_chassid.py` it defines the ground truth values the same way I do. 

There is an argument to be made for keeping with this (separate values) pattern for general consistency / maintainability for now and moving the whole chassisd test module to a more ideal format in the future.

I am not largely opinionated here though.